packages/cli/src/commands/election/activate.test.ts

```diff
@@ -91,6 +91,59 @@ testWithGanache('election:activate', (web3: Web3) => {
     )
   })
 
+  it('activate votes with --wait flag', async () => {
+    const kit = newKitFromWeb3(web3)
+    const [groupAddress, validatorAddress, userAddress] = await web3.eth.getAccounts()
+    const election = await kit.contracts.getElection()
+    const writeMock = jest.spyOn(ux.write, 'stdout')
+    const activateAmount = 12345
+    const logMock = jest.spyOn(console, 'log')
+
+    await setupGroupAndAffiliateValidator(kit, groupAddress, validatorAddress)
+    await registerAccountWithLockedGold(kit, userAddress)
+
+    await voteForGroupFrom(kit, userAddress, groupAddress, new BigNumber(activateAmount))
+
+    expect((await election.getVotesForGroupByAccount(userAddress, groupAddress)).active).toEqual(
+      new BigNumber(0)
+    )
+
+    await Promise.all([
+      testLocally(Activate, ['--from', userAddress, '--wait']),
+      new Promise<void>((resolve) => {
+        // at least the amount the --wait flag waits in the check
+        setTimeout(async () => {
+          await mineEpoch(kit)
+          resolve()
+        }, 1000)
```

Comment:
why 1 second specifically?

Reply:
just to match the wait time here https://github.com/celo-org/developer-tooling/blob/0ad9c011b868c4bf5456f4048cb6d405c9dd8c8e/packages/cli/src/commands/election/activate.ts#L45